src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs

```diff
@@ -603,6 +603,59 @@ public VariableScope[] GetVariableScopes(int stackFrameId)
             };
         }
 
+        internal void SetPathMappings(PathMapping[] pathMappings) => _pathMappings = pathMappings;
+
+        internal void UnsetPathMappings() => _pathMappings = null;
```

Comment:
Could this just be a property?

Reply:
It can, I think I started with just a normal property then went with a method when it seemed like I wanted to set some other stuff but that went away. Can definitely just use a property here if you wish.